app/controllers/application_controller.rb

```diff
@@ -2,5 +2,17 @@ class ApplicationController < ActionController::Base
 
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  protect_from_forgery with: :exception
+  # protect_from_forgery with: :exception
```

Comment:
We can remove this commented line if it's not being used

If you find yourself needing a regular controller (not API controller) you could create a new `ApiController` that will contain that line below with the `request.format.json?` and it inherits from this ApplicationController. That way, all of your API controllers will inherit from the ApiController (still have access to the ActionController and things defined here) and the `protect_from_forgery` will be set up for both types of controllers

_Not a required change, something to keep in mind_